appmgr/src/action/docker.rs

```diff
@@ -230,24 +231,24 @@ impl DockerAction {
         }
     }
 
-    pub fn uncontainer_name<'a>(name: &'a str) -> Option<(PackageId<&'a str>, Option<&'a str>)> {
```

Comment:
Fix for clippy

Reply:
![image](https://user-images.githubusercontent.com/2364004/148982541-29cd32ae-4c17-4140-8c1b-421cdaf0c202.png)

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#needless_lifetimes